### PR TITLE
GDB-8186: configure "infer" and "sameAs" depends on user settings

### PR DESCRIPTION
**Pull request by boyan-tonchev**
## What
The configuration "infer" and "sameAs" don't applied to "ontotext-yasgui-web-component" as default when open a new tab.

## Why
WB has such implementation.

## How
Updates version of "ontotext-yasgui-web-component". Adds configuration for "infer" and "sameAs" depends on user settings.

### CODE REVIEW

File: src/js/angular/sparql-editor/controllers.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ import 'services/ontotext-yasgui-web-component.service.js';
 import 'angular/externalsync/controllers';
 import {QueryType} from "../../../models/ontotext-yasgui/query-type";
 import {BeforeUpdateQueryResult, BeforeUpdateQueryResultStatus} from "../../../models/ontotext-yasgui/before-update-query-result";
+import Promise from "lodash/_Promise";
```
Comment (svilenvelikov) [inline src/js/angular/sparql-editor/controllers.js:17]: Interesting. Where did you saw this?

Comment (boyan-tonchev) [reply src/js/angular/sparql-editor/controllers.js]: This is auto import of IDE. I will remove it.

**Comment by boyan-tonchev** [comment]
Jenkins retry build

**Comment by boyan-tonchev** [comment]
Jenkis retry build

**Comment by boyan-tonchev** [comment]
Jenkins retry build

**Comment by boyan-tonchev** [comment]
Jenkins retry build

**Comment by boyan-tonchev** [comment]
Jenkins retry build

**Comment by boyan-tonchev** [comment]
Jenkins retry build

**Comment by boyan-tonchev** [comment]
Jenkins retry build

**Comment by boyan-tonchev** [comment]
Jenkins retry build

**Comment by boyan-tonchev** [comment]
Jenkins retry build

**Review by svilenvelikov — APPROVED** [review]

**Pull request merged**